nncf/quantization/algorithms/weight_compression/scale_estimation.py

```diff
@@ -365,7 +365,7 @@ def calculate_quantization_params(
         return result_scale, zp
 
     @staticmethod
-    def activations_to_wc_statistics(activations: List[Tensor]) -> WCTensorStatistic:
+    def activations_to_wc_statistics(activations: List[Tensor], transpose: bool) -> WCTensorStatistic:
```

Comment:
I'd pass `output_channel_axis` explicitly and re-implement definition of `reduction_shape` using this info.